.github/shiny-workflows/routine.sh

```diff
@@ -6,3 +6,8 @@ git commit R/sysdata.rda -m 'Resave data (GitHub Action)' || echo "No updates to
 # Update `Config/Needs/website` deployment dependencies
 Rscript ./.github/shiny-workflows/__update_website_deps.R
 git commit DESCRIPTION -m 'Update website deps (GitHub Action)' || echo "No new website vignette dependencies to commit"
+
+# Shrink images in man/figures
+sudo apt-get install optipng
+optipng man/figures/*.png
+git commit man/figures/*.png -m "Shrink images in man/figures (Github Action)" || echo "No changes to man/figures/*.png"
```

Comment:
Made a PR in rstudio/shiny-workflows#15

Reply:
If PR is approved, this code can be removed

Reply:
We're going to preemtively take this out hoping that rstudio/shiny-workflows#15 will merge soon (nothing _bad_ will happen in the meantime).